pixi.toml

```diff
@@ -80,7 +80,6 @@ pyarrow = "*"
 scikit-image = "*"
 scipy = "*"
 shapely = "*"
-streamz = ">=0.5.0"
```

Comment:
Was it installed and not used at all? I.e. are there no unit tests for `streamz`?

Reply:
Correct.